### PR TITLE
[JENKINS-60579] - Allow the usage of DescriptorVisibilityFilter to filter View properties on UI.

### PR DESCRIPTION
**Pull request by aHenryJard**
See [JENKINS-60579](https://issues.jenkins-ci.org/browse/JENKINS-60579).
Make it possible to have invisible properties on View that are persisted, in the same way that Jobs can have invisible properties.

### Proposed changelog entries

* Entry 1: Allow the usage of DescriptorVisibilityFilter to filter View properties on UI.

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### CODE REVIEW

File: core/src/main/java/hudson/model/View.java [hunk 1]
```diff
@@ -334,6 +335,14 @@ public void setDescription(String description) {
         return r;
     }
 
+    /**
+     * @return all the {@link ViewPropertyDescriptor}s that can be potentially configured on this View and are visible
+     * for the user. Use {@link DescriptorVisibilityFilter} to make a View property invisible for users.
+     */
```
Comment (oleg-nenashev) [inline core/src/main/java/hudson/model/View.java]: If it is not an override ...

```suggestion
     * @since TODO
     */
```

**Review by oleg-nenashev — APPROVED** [review]
Looks good to me, thanks!

**Comment by oleg-nenashev** [comment]
It might be a feature, not a bug. Will appreciate feedback from other reviewers

**Review by res0nance — APPROVED** [review]
LGTM 👍 , since it never worked I'd take it as a feature addition?

**Comment by oleg-nenashev** [comment]
I plan to merge it tomorrow if no negative feedback

File: core/src/main/java/hudson/model/View.java (hunk 1 above)
Comment (aHenryJard) [reply core/src/main/java/hudson/model/View.java]: done.

**Comment by oleg-nenashev** [comment]
Thanks @aHenryJard !

**Pull request merged**